### DISCUSSION TITLE
[NEW RELEASE] Introducing MARS5, open-source, insanely prosodic text-to-speech (TTS) model.

### DISCUSSION THREAD

**Discussion by akshhack** [openai/whisper#2215] [Show and tell]
![mars-banenr](https://github.com/openai/whisper/assets/21692676/31a12730-ee31-4f85-a37e-5f75c7d13834)

Hey community members! 👋 

We, at [CAMB.AI](https://www.camb.ai), are super stoked to announce the open source release of **MARS5**, a new speech emulation model that is able to replicate even extremely tough prosody like sports commentary, anime, movies with just a few seconds of audio reference. 

_Check out our release:_ https://github.com/Camb-ai/MARS5-TTS

Watch our demo here:

https://github.com/openai/whisper/assets/21692676/15672faf-6c60-4f12-a226-a19ad9d5f33d

and the full release video: https://www.youtube.com/watch?v=bmJSLPYrKtE

We're excited to hear feedback and see the community build on top of it!

Quick links:
**Discord:** https://discord.gg/ZzsKTAKM
**Github:** https://www.github.com/camb-ai/mars5-tts
**Website:** https://www.camb.ai/
**Youtube:** https://www.youtube.com/@camb-ai

**Comment by ejentos** [comment]
@akshhack is there any comparison with whisper?
